shenyu-admin/src/main/java/org/apache/shenyu/admin/service/impl/ApiServiceImpl.java

```diff
@@ -124,7 +128,7 @@ private String update(final ApiDTO apiDTO) {
             }
             if (ApiStateEnum.PUBLISHED.getState() == apiDO.getState()) {
                 register(apiDO);
-            } else if (ApiStateEnum.OFFLINE.getState() == apiDO.getState()) {
+            } else if (ApiStateEnum.UNPUBLISHED.getState() == apiDO.getState()) {
```

Comment:
The state judge should not change.

Reply:
For apidoc, the OFFLINE case is currently not implemented. In the next version, I plan to implement the OFFLINE function based on the application granularity of the apidoc module. The unregister() method should belong to the UNPUBLISHED case, not OFFLINE, so I modified the conditional judgment.

Reply:
Hi, @dragon-zhang , it has been adjusted. Is this okay?
![image](https://github.com/apache/shenyu/assets/13819823/f8c39973-90d9-44ee-a12c-de53bec688de)

Reply:
hi, @dragon-zhang .The original meaning of 'offline' has been preserved. Simultaneously adding optimization: When modifying apidoc, it supports deleting registered apis, but it is not supported when creating.